src/App.jsx

```diff
@@ -4,19 +4,19 @@ import "./App.css";
 import Home from "./Pages/Home";
 import Transaction from "./Pages/Transaction";
 import Settings from "./Pages/settings";
+import AuthPage from "./Pages/AuthPage";
+
 
 function App() {
   return (
     <div className="app">
-
       <Routes>
+        <Route path="/authpage" element={<AuthPage />} />
```

Comment:
Move this route below `<Route path="/" element={<Home />} />`
Observe that nothing happens when you click the google or apple icon right now